README.md

```diff
@@ -106,12 +106,14 @@ optional flags:
     	A comma separated list of file types to ignore
   -groupby string
     	Group output by filetype, directory, pass-fail. Supported for Standard and JSON reports
-  -output string
-    	Destination to a file to output results
   -quiet
     	If quiet flag is set. It doesn't print any output to stdout.
   -reporter string
-    	Format of the printed report. Options are standard and json (default "standard")
+		A string representing report format and optional output file path separated by colon if present.
+		Usage: --reporter <format>:<optional_file_path>
+		Multiple reporters can be specified: --reporter json:file_path.json --reporter junit:another_file_path.xml
+		Omit the file path to output to stdout: --reporter json or explicitly specify stdout using "-": --reporter json:-
+		Supported formats: standard, json, junit (default: "standard")
```

Comment:
Too much indentation

Reply:
How do you want me to resolve this? Should I put everything on a single line?